stdlib/public/core/UnsafePointer.swift

```diff
@@ -370,7 +370,7 @@ public struct UnsafePointer<Pointee>: _Pointer {
   ///            by the key path, or `nil`.
   @inlinable
   @_alwaysEmitIntoClient
-  @_unavailableInEmbedded
+  @_transparent
```

Comment:
since this is applicable for both embedded and non-embedded systems what impact does that have per ABI?
@Azoy do you have any objection here?

Reply:
Yeah, I'm not sure we'd want these things to be `@_transparent` in the non-embedded version. That seems a bit too aggressive for this API. If you truly need this for embedded and can't just rely on the `@inlinable`/`@_alwaysEmitIntoClient` declarations, then I'd suggest guarding this with the feature flag. In terms of ABI, this shouldn't be too big of a concern because it's already visible to clients.

Reply:
Added the #if $Embedded guards.